02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,55 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+  	<link type="text/css" rel="stylesheet" href="css/style.css"/>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
 </head>
 <body>
-
+  <h1>IAMDIANA</h1>
+  <img src="https://scontent-dfw1-1.xx.fbcdn.net/v/t1.0-0/p206x206/1465342_10202203994676888_1874833479_n.jpg?oh=a3f03ebf7439d902ba10e7327bcf03c4&oe=58374AA1">
+<hr></hr>
+<p id="aboutme"> Hello! I am from San Antonio, TX and a UTSA alumni.  I lived in Brooklyn for a couple of years where I worked as a tutor.  I currently work in ecommerce for San Antonio Shoemakers.  I am also attending Austin Coding Academy to learn a new skill. Thanks for visiting. </p>
+<div id="first">
+  <or>
+  <p> Interests</p>
+    <li>food</li>
+    <li>music</li>
+    <li>family</li>
+    <li>travel</li>
+  </or>
+</div>
+<div id="second">
+    <or>
+      <p>Favorites</p>
+        <li>Amelie</li>
+        <li>Bread</li>
+        <li>Scary Movies</li>
+        <li>Dogs</li>
+    </or>
+  </div>
+  <table>
+<thead>
+    <tr>
+      <th colspan="3">Me in Pictures</th>
+        </tr>
+ </thead>
+    <tbody>
+    <tr>
+        <td><img src="https://scontent-dfw1-1.xx.fbcdn.net/v/t1.0-9/11737962_1654912098063680_8943687515255524354_n.jpg?oh=e047ae13845eb3a28241ce88b4bce294&oe=58373341"/></td>
+        <td><a href="http://www.utsa.edu/"><img src="http://www.cs.utsa.edu/~qitian/utsa.jpg"></a></td>
```

Comment:
Adding `target="_blank"` to your `<a>` tags will make the link open in a new tab, allowing the user to stay on your site (which is what we want, after all, right?!).